### PR TITLE
feature/bullet-list

### PR DESCRIPTION
**Pull request by seesleestak**

### CODE REVIEW

File: packages/bullet-list/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# hixme-ui bullet-list
+
+```bash
+npm i --save @hixme-ui/bullet-list
+```
+[View it live](https://hixme.github.io/hixme-ui/bullet-list)
```
Comment (billyxs) [inline packages/bullet-list/README.md]: want to remove this until we have a hui-pages example of it?

Comment (seesleestak) [reply packages/bullet-list/README.md]: I'll make up a docs page for it real quick.

File: packages/hui-pages/src/routes/BulletListDocs/index.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+import BulletList from '@hixme-ui/bullet-list'
+
+import { Code, DocsTitle } from '../../components'
+
+const BulletListDocs = () => {
+  const myUnorderedList = [
+    'One item',
```
Comment (billyxs) [inline packages/hui-pages/src/routes/BulletListDocs/index.js]: should be One Fish, Two Fish, Red Fish, Blue Fish

Comment (seesleestak) [reply packages/hui-pages/src/routes/BulletListDocs/index.js]: That was the idea. Eh what the heck, I can have a fish list.

**Comment by seesleestak** [comment]
Thanks @pixelfuture!

File: packages/bullet-list/README.md [hunk 3]
```diff
@@ -11,10 +11,10 @@ npm i --save @hixme-ui/bullet-list
 import BulletList from '@hixme-ui/bullet-list'
 
 const myUnorderedList = [
-  'One item',
-  'Two item',
-  'Red item',
-  'Blue item',
+  'One fish',
```
Comment (billyxs) [inline packages/bullet-list/README.md]: haha...you are a great sport

Comment (seesleestak) [reply packages/bullet-list/README.md]: `cgn`, easy

Comment (billyxs) [reply packages/bullet-list/README.md]: aww yeah

**Pull request merged**